### PR TITLE
Customer Membership Dialog Tidy Up Styles

### PR DESCRIPTION
**Pull request by blubaugha**
### Summary
Just tidying up some of the membership styles in the customer details dialog.

### Screenshots
**Desktop**
desktop

**iPad Pro - Landscape**
ipadpro-landscape

**iPad Pro - Portrait
ipadpro-portrait

**iPad - Landscape**
ipad-landscape

**iPad - Portrait**
ipad-portrait

### CODE REVIEW

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                             {{screen.membershipLabel}}
                         </div>
                         <div *ngIf="screen.customer.memberships && screen.customer.memberships.length">
-                            <span *ngFor="let membership of screen.customer.memberships" class="item">
+                            <span *ngFor="let membership of screen.customer.memberships" class="item" responsive-class>
```
Comment (jchirico8) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.html:30]: This page was as responsive as you on slack when you're testing add customer!

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.scss [hunk 2]
```diff
@@ -128,6 +112,7 @@ app-content-card {
 
 .membership-icon{
   grid-area: membership-icon;
+  margin-top: -10px; // Force top align because icon height and width are not equal
```
Comment (jchirico8) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.scss:115]: 🤮

**Review by jchirico8 — APPROVED** [review]

**Review by chenson42 — APPROVED** [review]

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.html (hunk 1 above)
Comment (blubaugha) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.html:30]: # HAHA DUDE YEAH!

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.scss (hunk 2 above)
Comment (blubaugha) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/sale/customer-details-dialog/customer-details-dialog.component.scss:115]: # DAMN, NOW THERE'S PUKE ON THE FILE!

**Pull request merged**